src/main/services/uml-element/uml-element.ts

```diff
@@ -73,6 +73,7 @@ export abstract class UMLElement implements IUMLElement, ILayoutable {
   strokeColor?: string;
   textColor?: string;
   assessmentNote?: string;
+  resizeFrom = '';
```

Comment:
I would propose to create an enum for this containing the four elements 
TOP_RIGHT,
TOP_LEFT,
BOTTOM_RIGHT,
BOTTOM_LEFT.

It is a bit more descriptive and overall better pratice.